### DISCUSSION TITLE
我们是否需要添加允许列表？还是阻止列表已经排除了允许列表中的项目？我使用的是Pihole，所以我应该使用哪些列表？

### DISCUSSION THREAD

**Discussion by AyanokojiK1yotaka** [Potterli20/hosts#22] [General]
我们是否需要添加允许列表？还是阻止列表已经排除了允许列表中的项目？我使用的是Pihole，所以我应该使用哪些列表？

**Comment by Potterli20** [comment]
白名单也有，但是pihole的白名单不支持链接的，我记得是要写脚本才行

**Comment by Potterli20** [comment]
hosts.trli.club里，有个allow的白名单

**Reply by AyanokojiK1yotaka** [reply]
白名单中的域名已经从黑名单中移除了吗？

**Reply by Potterli20** [reply]
> 白名单中的域名已经从黑名单中移除了吗？

没有没有，也是在黑名单上处理了。但以防万一，还是单独白名单来

**Reply by Potterli20** [reply]
不对，我没有睡醒。
应该是部分误杀的已经从黑名单上移除

**Comment by Potterli20** [comment]
建议和白名单配合用
